src/install-choose/install-choose-dialog.ts

```diff
@@ -480,6 +480,9 @@ class ESPHomeInstallChooseDialog extends LitElement {
       svg {
         fill: currentColor;
       }
+      .svg-filter {
```

Comment:
Move this to `esphomeDialogStyles`. Let's also rename it to `svg-fill-primary`

Reply:
I opted to create a new `esphomeSvgStyles` as there was use in the web installer that was not in a dialog